literature.bib

```diff
@@ -326,6 +326,17 @@ @mastersthesis{Guething23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2024 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{KCP+:SA24,
+	author  = {Sayash Kapoor and Emily M. Cantrell and Kenny Peng and Thanh Hien Pham and Christopher A. Bail and Odd Erik Gundersen and Jake M. Hofman and Jessica Hullman and Michael A. Lones and Momin M. Malik and Priyanka Nanayakkara and Russell A. Poldrack and Inioluwa Deborah Raji and Michael Roberts and Matthew J. Salganik and Marta Serra-Garcia and Brandon M. Stewart and Gilles Vandewiele and Arvind Narayanan},
+	title   = {{REFORMS: Consensus-based Recommendations for Machine-learning-based Science}},
```

Comment:
capizalizemytitle.com says `REFORMS: Consensus-Based Recommendations for Machine-Learning-Based Science` to me

Reply:
good catch